res/openvpn-gui-res-dk.rc

```diff
@@ -149,49 +149,49 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_DANISH, SUBLANG_DEFAULT
 BEGIN
     GROUPBOX "Brugergrænseflade", 201, 6, 12, 235, 30
-    LTEXT "Language:", ID_TXT_LANGUAGE, 17, 25, 52, 12
+    LTEXT "Sprog:", ID_TXT_LANGUAGE, 17, 25, 52, 12
     COMBOBOX ID_CMB_LANGUAGE, 57, 23, 171, 400, CBS_DROPDOWNLIST | WS_TABSTOP
-    GROUPBOX "Startup", 202, 6, 47, 235, 30
-    AUTOCHECKBOX "Launch on Windows startup", ID_CHK_STARTUP, 17, 59, 200, 12
+    GROUPBOX "Autostart", 202, 6, 47, 235, 30
+    AUTOCHECKBOX "Start med Windows", ID_CHK_STARTUP, 17, 59, 200, 12
 
-    GROUPBOX "Preferences", 202, 6, 82, 235, 90
-    AUTOCHECKBOX "Append to log", ID_CHK_LOG_APPEND, 17, 95, 60, 10
-    AUTOCHECKBOX "Show script window", ID_CHK_SHOW_SCRIPT_WIN, 17, 110, 200, 10
-    AUTOCHECKBOX "Silent connection", ID_CHK_SILENT, 17, 125, 200, 10
-    LTEXT "Show Balloon", ID_TXT_BALLOON, 17, 140, 100, 10
-    AUTORADIOBUTTON "On connect", ID_RB_BALLOON1, 28, 155, 50, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "On connect/reconnect", ID_RB_BALLOON2, 86, 155, 90, 10
-    AUTORADIOBUTTON "Never", ID_RB_BALLOON0, 181, 155, 40, 10
+    GROUPBOX "Indstillinger", 202, 6, 82, 235, 90
+    AUTOCHECKBOX "Tilføj til log", ID_CHK_LOG_APPEND, 17, 95, 60, 10
+    AUTOCHECKBOX "Vis script vindue", ID_CHK_SHOW_SCRIPT_WIN, 17, 110, 200, 10
+    AUTOCHECKBOX "Stille forbindelse", ID_CHK_SILENT, 17, 125, 200, 10
+    LTEXT "Vis Popup", ID_TXT_BALLOON, 17, 140, 100, 10
+    AUTORADIOBUTTON "Ved forbind", ID_RB_BALLOON1, 28, 155, 50, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Ved forbind/genforbind", ID_RB_BALLOON2, 86, 155, 90, 10
+    AUTORADIOBUTTON "Aldrig", ID_RB_BALLOON0, 181, 155, 40, 10
 END
 
 /* Advanced Dialog */
 ID_DLG_ADVANCED DIALOGEX 6, 18, 252, 218
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
-CAPTION "Advanced"
+CAPTION "Avanceret"
 FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_DANISH, SUBLANG_DEFAULT
 BEGIN
-    GROUPBOX "Configuration Files", 201, 6, 12, 235, 45
-    LTEXT "Folder:", ID_TXT_FOLDER, 17, 25, 32, 10
-    LTEXT "Extension:", ID_TXT_EXTENSION, 17, 40, 52, 10
+    GROUPBOX "Configurations Filer", 201, 6, 12, 235, 45
+    LTEXT "Mappe:", ID_TXT_FOLDER, 17, 25, 32, 10
+    LTEXT "Endelse:", ID_TXT_EXTENSION, 17, 40, 52, 10
     EDITTEXT ID_EDT_CONFIG_DIR, 53, 23, 150, 12, ES_AUTOHSCROLL
     EDITTEXT ID_EDT_CONFIG_EXT, 53, 38, 25, 12, ES_AUTOHSCROLL
-    PUSHBUTTON "…", ID_BTN_CONFIG_DIR, 208, 23, 25, 12
+    PUSHBUTTON "...", ID_BTN_CONFIG_DIR, 208, 23, 25, 12
```

Comment:
Why are you translating this?

Reply:
This one i had some problems with, because what to use in danish, you could use the "Vælge","Valg" or you could use "Åben". thats why i just kept it at the ... like the orginal.

It's the for browse button beside the filepaths in the settings advanced tab:
![image](https://user-images.githubusercontent.com/27216403/35278002-f0105374-0048-11e8-880a-78440e90de69.png)

Reply:
What about `"Vælg sti..."` (`"Choose path..."`)?